src/plugins/htmlVideoPlayer/plugin.js

```diff
@@ -390,6 +407,68 @@ function tryRemoveElement(elem) {
             });
         }
 
+        /**
+         * @private
+         */
+        onShakaError(error) {
+            console.error('shaka: error code', error.code, 'object', error);
+        }
+
+        /**
+         * @private
+         * @param e {Event} The event received from the `<video>` element
+         */
+        onShakaErrorEvent = (e) => {
+            // Extract the shaka.util.Error object from the event.
+            this.onShakaError(e.detail);
+        };
```

Comment:
I think we should stop playback or something.
I got an error when trying to play a video with an audio track longer than the video track.

For HLS.js this is called (as well as for HtmlMediaElement):
https://github.com/jellyfin/jellyfin-web/blob/073b7c6d697a16894998025e4d3be5a2abd7d2ce/src/plugin/htmlVideoPlayer/plugin.js#L1002